pkg/handlers/generic/lifecycle/csi/nutanix/handler.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2024 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package nutanix
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/spf13/pflag"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/api/v1alpha1"
+	"github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pkg/handlers/generic/lifecycle/config"
+	csiutils "github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pkg/handlers/generic/lifecycle/csi/utils"
+	"github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pkg/handlers/options"
+	handlersutils "github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pkg/handlers/utils"
+)
+
+const (
+	defaultStorageHelmReleaseName      = "nutanix-csi"
+	defaultStorageHelmReleaseNamespace = "ntnx-system"
```

Comment:
you mean to move it to `kube-systems` namespace?

Reply:
I wanted to but can't - see https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/733#discussion_r1648110996

Reply:
The chart doesn't support that atm https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/733#discussion_r1648110996

Reply:
Thanks